$NetBSD: patch-aa,v 1.1.1.1 2003/05/23 14:44:27 seb Exp $

This patch had been manually stripped to keep only the
pertinent part.

--- doc/scli.info.orig	2003-05-23 13:16:24.000000000 +0000
+++ doc/scli.info
@@ -1,4 +1,9 @@
-This is scli.info, produced by makeinfo version 4.2 from scli.texinfo.
+This is scli.info, produced by makeinfo version 4.5 from scli.texinfo.
+
+INFO-DIR-SECTION Network utilities
+START-INFO-DIR-ENTRY
+* Scli: (scli).                 SNMP command line interface.
+END-INFO-DIR-ENTRY
 
    This file documents the `scli' SNMP command line interface.
 
